packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -80,7 +230,11 @@ export const ReleaseModal = ({
                 <Button
                   name="submit"
                   loading={isLoading}
-                  disabled={!values.name || values.name === initialValues.name}
+                  disabled={
+                    !dirty ||
+                    !values.name ||
+                    (values.isScheduled && (!values.time || !values.date || !values.timezone))
```

Comment:
i'm not sure about this to be honest, i think it's quite bad UX because user's can submit their form and find out what the issue is, they're just gonna be confused. Why are we not using the validation to give feedback?

Reply:
I have same opinion as you, but last time I implemented error state and I was asked to revert it.
https://www.figma.com/file/MWPKHNT98LLWHGRp3MyWqu?node-id=1339:17473&mode=design#643292761

Reply:
On the contrary, when I try to use error state on these fields which are all select DD, its of no use as inputs are always going to be blank on wrong entry, and so we disable Save button.

Reply:
Based on what discussed on Slack I think you can safely remove this 🎉